test/ModifyLiquidity.t.sol

```diff
@@ -0,0 +1,139 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.20;
+
+import {Test} from "forge-std/Test.sol";
+import {Deployers} from "./utils/Deployers.sol";
+import {GasSnapshot} from "forge-gas-snapshot/GasSnapshot.sol";
+import {PoolKey} from "src/types/PoolKey.sol";
+import {IPoolManager} from "src/interfaces/IPoolManager.sol";
+import {IHooks} from "src/interfaces/IHooks.sol";
+import {Position} from "src/libraries/Position.sol";
+import {PoolId} from "src/types/PoolId.sol";
+
+contract ModifyLiquidityTest is Test, Deployers, GasSnapshot {
+    PoolKey simpleKey; // vanilla pool key
```

Comment:
could just use the `PoolKey key` thats defined in Deployers.sol already I think

Reply:
i think that key is only set with `initializeManagerRoutersAndPoolsWithLiq` so I just defined a new key